services/QuillLMS/app/models/accessibility_setting.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: accessibility_settings
+#
+#  id                :bigint           not null, primary key
+#  snackbar_duration :integer
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#  user_id           :integer          not null
+#
+class AccessibilitySetting < ApplicationRecord
+  belongs_to :user
+
+  validates :user, presence: true
```

Comment:
Some potential validations:

validates :snackbar_duration, presence: true, numericality: { greater_than: 0 }
validates :user_id, uniqueness: true. (<- this could also be db constraint)

Reply:
I'm disinclined to put a `presence: true` validation on `snackbar_duration`, because I think we'll have to remove it in order to extend this table, and my plan is still to just default to the current timeout on the frontend if there's no value set, so it's not strictly necessary to have a value stored here. It is currently the case, however, that it doesn't really make sense to have this record if `snackbar_duration` doesn't exist, so let me know if you think it would be better to add it here and we can just remove it down the line if necessary.

Reply:
That makes sense.  I would leave it out then.